packages/core/core-flows/src/cart/workflows/add-shipping-method-to-cart.ts

```diff
@@ -112,8 +112,8 @@ export const addShippingMethodToWorkflow = createWorkflow(
 
     updateTaxLinesWorkflow.runAsStep({
       input: {
-        cart_or_cart_id: input.cart_id,
-        shipping_methods: shippingMethodsToAdd,
+        cartId: input.cart_id,
+        shippingMethods: shippingMethodsToAdd,
```

Comment:
nit(non blocking): havent we settle a convention for input prop naming to be snake cases?

Reply:
Oh yeah, my bad. Will update